integration/tests/managed/managed_nodegroup_test.go

```diff
@@ -487,15 +419,20 @@ var _ = Describe("(Integration) Create Managed Nodegroups", func() {
 					return fmt.Sprintf("%s.%s", serverVersion.Major, strings.TrimSuffix(serverVersion.Minor, "+"))
 				}, k8sUpdatePollTimeout, k8sUpdatePollInterval).Should(Equal(nextVersion))
 
-				By(fmt.Sprintf("upgrading nodegroup %s to Kubernetes version %s", initialNodeGroup, nextVersion))
-				cmd = params.EksctlUpgradeCmd.WithArgs(
-					"nodegroup",
-					"--verbose", "4",
-					"--cluster", params.ClusterName,
-					"--name", initialNodeGroup,
-					"--kubernetes-version", nextVersion,
-				)
-				Expect(cmd).To(RunSuccessfullyWithOutputString(ContainSubstring("nodegroup successfully upgraded")))
+				upgradeNg := func(ngName string) {
```

Comment:
Look at you going all functiony

![homerooo](https://user-images.githubusercontent.com/182850/150857680-a50cb9b7-e18c-4ed2-bb93-a98d9bbd2151.jpeg)

Reply:
😂😂😂😂 was that or for loop . For loop felt wrong